lib/cdo/shared_constants/pd/teacher1819_application_constants.rb

```diff
@@ -0,0 +1,127 @@
+module Teacher1819ApplicationConstants
+  # Remove newlines and leading whitespace from multiline strings
+  def self.clean_multiline(string)
+    string.gsub(/\n\s*/, ' ')
+  end
+
+  SECTION_HEADERS = {
+    section_1_about_you_and_your_school: 'About You and Your School',
+    section_2_choose_your_program: 'Section 2: Choose Your Program',
+    section_3_summer_workshop: 'Section 3: Summer Workshop',
+    section_4_submission: 'Section 4: Submission'
+  }
+
+  PAGE_LABELS = {
+    section_1_about_you_and_your_school: {
+      title: 'Title',
+      first_name: 'First Name',
+      preferred_first_name: 'Preferred First Name',
+      last_name: 'Last Name',
+      account_email: 'Account Email',
+      alternate_email: 'If you use another email, enter it here:',
+      phone: 'Phone',
+      address: 'Home Address',
+      city: 'City',
+      state: 'State',
+      zip_code: 'Zip Code',
+      gender_identity: 'Gender Identity',
+      race: 'Race',
```

Comment:
Is this a (Select all that apply) as well?

Reply:
Yes, will add in a follow up